core/consensus/aura/slots/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "substrate-consensus-aura-slots"
+version = "0.1.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+description = "Generic slots-based utilities for consensus"
+edition = "2018"
+
+[dependencies]
+parity-codec = "3.0"
+client = { package = "substrate-client", path = "../../../client" }
+primitives = { package = "substrate-primitives", path = "../../../primitives" }
+runtime_primitives = { package = "sr-primitives", path = "../../../sr-primitives" }
+aura_primitives = { package = "substrate-consensus-aura-primitives", path = "../primitives" }
+consensus_common = { package = "substrate-consensus-common", path = "../../common" }
+inherents = { package = "substrate-inherents", path = "../../../inherents" }
+futures = "0.1.17"
+tokio = "0.1.7"
```

Comment:
Latest is `0.1.15`, `0.1` should be okay.

Reply:
(not true -- we require features after 0.1)